source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,300 @@
+
+- **Kind**: Secret Text
+- **Scope**: Global
+- **Secret**: Paste the GitHub token you created.
+- **ID**: blank
+- **Description**: GitHub
+
+Create another for a Terminus Token, which can either be an existing token or [create a new token](https://pantheon.io/docs/machine-tokens) from your user dashboard for Jenkins.
+
+- **Kind**: Secret Text
+- **Scope**: Global
+- **Secret**: Paste the Terminus token you created.
+- **ID**: blank
+- **Description**: Terminus
+
+After saving both, your credentials will be accessible for secure use.
+
+![Credentials Page](/source/docs/assets/images/integrations/jenkins/2_credentials.png)
+
+4. Return to the main Jenkins dashboard and select **Manage Jenkins**, then **Configure System**.
+
+5. Scroll to the **GitHub Servers** section  and click on **Add GitHub Server**.
+
+6. Leave the **API URL** as "https://api.github.com." For **Credentials**, click **Add** then from the drown-down you can select your new GitHub credentials. Then **Save**.
+
+7. Click ```Test Connection``` and you should see the GitHub username which created the token.
+
+8. Click Save.
+
+## Create Jenkins Project
+
+## Configure Jenkins
+
+### Create the Jenkins Project
+
+1. Log into the Jenkins dashboard as an admin user. Click on **New Item**.
+
+    ![Jenkins dashboard](/source/docs/assets/images/integrations/jenkins/jenkins_dash.png)
+
+2. Give the project a name with no spaces. Select **Freestyle Project** and click **OK** to save.
+
+    ![New Jenkins project](/source/docs/assets/images/integrations/jenkins/new_job.png)
+
+3. The next page lets you configure options for this project. In the **General** tab, select "GitHub project" and enter the repository URL (e.g. `https://github.com/YOUR-ORG/YOUR-PROJECT`).
+
+4. In the **Source Code Management** tab select **Git**, then **Advanced**. Add the following information:
+
+    - **Repository URL**: The path to the .git file of your repository, e.g. `https://github.com/YOUR-ORG/YOUR-PROJECT.git`
+    - **Name**: `origin`
+    - **Refspec**: blank
+    - **Branch specifier**: `origin/*`
+    - **Additional Behaviours**: Add **Prune stale remote-tracking branches**:
+
+    ![SCM view](/source/docs/assets/images/integrations/jenkins/scm_settings.png)
+
+5. We want code changes to trigger our build (as opposed to setting up a periodic build, for example). Under **Build Triggers**, Check the box labelled, "GitHub hook trigger for GITScm polling".
+
+6. Under **Build Environment**, check the box labelled "Inject environment variables to the build process."
+
+
+7. In the **Properties Content** field, add the following variable one per line, with no quotation marks:
+
+    - **SITE_ID=**your-site-name:
+
+    ![Env vars view](/source/docs/assets/images/integrations/jenkins/env_vars.png)
+
+8. Check "Use secret text(s) or file(s)" option, and add a secret text binding. Name it **TERMINUS_TOKEN** and select the Terminus secret text credential from the dropdown.
```

Comment:
This no longer applies to this section, correct? If so, I'll remove it during my edits.